### PR TITLE
Quotient action for Grassmann

### PR DESCRIPTION
**Pull request by mateuszbaran**
I've implemented the action that defines the Grassmann manifold as a quotient to complete the implementation.

### CODE REVIEW

**Review by kellertuer — APPROVED** [review]
Nice!

**Comment by kellertuer** [comment]
and from the last commit you might even have fixed code coverage for the two things I mentioned? That would be very cool :) Slowly working towards more coverage!

**Comment by mateuszbaran** [comment]
I actually don't think this will cover any uncovered lines. And I'm not sure if I should call it `MatrixColumnwiseMultiplicationAction` or `MatrixRowwiseMultiplicationAction`, given that it multiplies transpositions of each row?

**Comment by kellertuer** [comment]
> I actually don't think this will cover any uncovered lines. And I'm not sure if I should call it `MatrixColumnwiseMultiplicationAction` or `MatrixRowwiseMultiplicationAction`, given that it multiplies transpositions of each row?

But isn't it actually (just) the application of the rotation (or the unitary matrix) to the rows of a matrix (due to transpose)? I find the name a little clumsy, why not just ? `RowwiseMultiplicationAction`? The the `MultiplicationAction` we also do not include the matrix.

File: src/groups/rotation_action.jl [hunk 1]
```diff
@@ -129,3 +129,60 @@ end
...
+###
+
+@doc raw"""
+    MatrixColumnwiseMultiplicationAction(
+        M::AbstractManifold,
+        On::GeneralUnitaryMultiplicationGroup,
+        AD::ActionDirection = LeftAction(),
+    )
+
+Space of actions of the (special) unitary or orthogonal group [`GeneralUnitaryMultiplicationGroup`](@ref)
```
Comment (kellertuer) [inline src/groups/rotation_action.jl:142]: While checking for the name – the type is called action, but the documentation says its the space of the action., I think it is the second? Also the docs could include the subtype info (in line140)

**Comment by mateuszbaran** [comment]
> But isn't it actually (just) the application of the rotation (or the unitary matrix) to the rows of a matrix (due to transpose)? I find the name a little clumsy, why not just ? `RowwiseMultiplicationAction`? The the `MultiplicationAction` we also do not include the matrix.

Yes, `RowwiseMultiplicationAction` is a better name :+1: .

File: src/groups/rotation_action.jl (hunk 1 above)
Comment (mateuszbaran) [reply src/groups/rotation_action.jl:142]: So, I'm not sure -- do you prefer the "action" or "space of actions" name?

I'll include the subtype info and move the constructor signature below.

Comment (kellertuer) [reply src/groups/rotation_action.jl]: I just felt that, when the type is called action, the docs should not say its the space of actions?

Comment (mateuszbaran) [reply src/groups/rotation_action.jl]: OK, I see, I'll remove "Space of" then.

**Comment by mateuszbaran** [comment]
So I think this is ready to be merged now?

**Pull request merged**